MatrixKit/Utils/MXKEventFormatter.m

```diff
@@ -1362,6 +1362,8 @@ - (UIColor*)textColorForEvent:(MXEvent*)event
                     textColor = _defaultTextColor;
                 }
                 break;
+            case MXEventSentStateEncrypting:
```

Comment:
Events in encrypting state should be in a another color (green in case of Riot).